pages/download.html

```diff
@@ -309,6 +309,20 @@ <h3>{% trans "Linux / Source Code" %}</h3>
   </div>
 </div>
 
+<div id="snapshots" class="docs-section container">
+  <h2><a class="headline-link" href="#snapshots">{% blocktrans %}Development Snapshots{% endblocktrans %}</a></h2>
+  <p>
+    {% blocktrans with source_link="https://github.com/mixxxdj/mixxx" windows_snapshot_link="https://downloads.mixxx.org/builds/appveyor/cmake_build/" macos_snapshot_link="https://downloads.mixxx.org/builds/travis/main/" ubuntu_snapshot_link="https://downloads.mixxx.org/builds/travis/main/" %}Want to test and experiment with the latest changes?
+    The best way to keep up with Mixxx development is to grab the <a href="{{ source_link }}">source code</a> and compile Mixxx yourself.
+    Additionally, we also provide development snapshots for <a href="{{ windows_snapshot_link }}">Windows</a>,  <a href="{{ macos_snapshot_link }}">macOS</a> and <a href="{{ ubuntu_snapshot_link }}">Ubuntu</a>.{% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}Some developers choose to use the main branch for performance and new features.
```

Comment:
This wording could be misread as "performance" as in "more efficient software".
```suggestion
    {% blocktrans %}Some developers choose to use the main branch to perform using the latest features.
```

Reply:
Hmm, this sounds like we consider the `main` branch as stable enough to perform gigs, which is mostly the case but not always. I'll just remove the sentence, because it's redundant with the first sentence in the paragraph above.